test/core.jl

```diff
@@ -2450,7 +2451,7 @@ end
 # pull request #9534
 @test_throws BoundsError((1, 2), 3) begin; a, b, c = 1, 2; end
 let a = []
-    @test_broken try; a[]; catch ex; (ex::BoundsError).a === a && ex.i == (1,); end # TODO: Re-enable after PLI
+    @test try; a[]; catch ex; (ex::BoundsError).a === a && ex.i == (); end
```

Comment:
Yes! This is correct.  I don't doubt that we used to throw a BoundsError that had a implicitly generated linear `1`, but this is better.